docs/rfcs/0004-suggested-excluded-apps-tech-spec.md

```diff
@@ -0,0 +1,139 @@
+-   (8-13 points) Deep linking into app for notification to work ([VPN-5034](https://mozilla-hub.atlassian.net/browse/VPN-5034)): Currently, our navigation code allows us to switch tabs, but does not permit moving to a deeper screen - so we can move a user to the settings screen, but can't move them into a child menu or child's child menu within the screen (like App Exclusions). We need deep linking to be able to open the notification where we want.
+-   (2-5 points) Create initial list of recommended app exclusions
+
+### Create daily task in app (9-13 points total)
+<img src="./images/0004-02.png" width=800 alt="New daily task">
+
+-   (3-5 points) Daily task to check for suggested apps that are installed on the device (after one week from first install), halt task if don't have notification permissions or if the platform doesn't have app exclusions feature. (Create dev option to immediately pull down the new list.)
+    -   This will be run by the app for desktop, and a [PeriodicWorkRequest](https://developer.android.com/reference/androidx/work/PeriodicWorkRequest) on Android. (iOS does not have split tunnel.)
+        -   After running the job, we will create a 24 hour countdown timer to run the job next. Additionally, we'll record the time for this job as a setting, and we'll check this timestamp on app/daemon launch - if more than 24 hours have passed we'll
+    -   Create a dev option to pull a new suggested apps list immediately.
+-   (3-5 points) Is there one or more new apps?
+    -   Check against setting - if we have ever sent a notification for this app before, don't send one (this should be done off the app's domain name or something, not marketing name - that can change)
+    -   Pop an OS-level notification (on Android, use a suggestion channel). The text is intentionally generic, and makes sense if there is one new app or multiple.
+    -   Add new app(s) to "have sent notification" list
+    -   (Create a dev option to clear the "have sent notification" list. Create another dev option to immediately run the daily check job.)
+-   (1 point) Add this notification type to the settings screen that allows per-type opt out (App permissions -> Notifications)
+-   (2 points) Ensure tapping notification leads to correct screen
+
+### Improve existing split tunnel screen (10-16 points total)
+[Figma mocks](https://www.figma.com/file/UZYzma7hlcfE5ke3z8jGbN/App-exclusions-suggestions?type=design&node-id=196-6366&mode=design&t=RL1hdfBQLMS1rKVa-0)
+
+<img src="./images/0004-03.png" width=500 alt="Updated flow for existing App Exclusion screen">
+
+-   (2 points) Run the task on the launch of screen
+    -   Update the "have seen these apps" settings list with all suggested apps that are on the system. (This means that if a new app is seen on the Settings screen before the daily job runs and we send a notification, we never send a notification. This was confirmed by Santiago.)
```

Comment:
That is correct, adding some clarification. (And remember that while the notification is popped because of a new app or apps, that app (or apps) are never mentioned by name in the notification to the user.)